trieste/objectives/single_objectives.py

```diff
@@ -93,6 +114,24 @@ def scaled_branin(x: TensorType) -> TensorType:
 """ The search space for the :func:`branin` function. """
 
 
+Branin = SingleObjectiveTestProblem(
+    objective=branin,
+    search_space=BRANIN_SEARCH_SPACE,
+    minimizers=BRANIN_MINIMIZERS,
+    minimum=BRANIN_MINIMUM,
```

Comment:
Prefer to keep them both to avoid breaking other people's code and to avoid wasting time updating our unit tests. The cost of having them is minimal.

Reply:
Done (since you clearly care about this way more than I do). Please review all the files very carefully, as with so much copy and pasting there is a good chance of typos.

Reply:
Thank you @uri-granta :)